build.install4j

```diff
@@ -417,7 +588,7 @@
           </screen>
         </screens>
       </application>
-      <application name="" id="uninstaller" customizedId="" beanClass="com.install4j.runtime.beans.applications.UninstallerApplication" enabled="true" commentSet="false" comment="" actionElevationType="none" fileset="" customIcnsFile="" customIcoFile="" macEntitlementsFile="" automaticLauncherIntegration="false" launchMode="startupFirstWindow" launchInNewProcess="false" launchSchedule="updateSchedule" allLaunchers="true">
+      <application name="" id="uninstaller" customizedId="" beanClass="com.install4j.runtime.beans.applications.UninstallerApplication" enabled="true" commentSet="false" comment="" actionElevationType="elevated" fileset="" customIcnsFile="" customIcoFile="" macEntitlementsFile="" automaticLauncherIntegration="false" launchMode="startupFirstWindow" launchInNewProcess="false" launchSchedule="updateSchedule" allLaunchers="true">
```

Comment:
I built the Windows x64 installer with this [patch](https://github.com/triplea-game/triplea/files/1045703/pr-1783-protocol-handler-registry-keys.diff.txt) and installed it as a non-admin user.  I now see the `triplea` key and all its values under `HKCR`.  After entering the URL `triplea:test` into Firefox, I was prompted to run TripleA.  I observed no errors in the log.